src/control/cmd/dmg/auto.go

```diff
@@ -64,31 +68,41 @@ func (cmd *configGenCmd) Execute(_ []string) error {
 	}
 	switch cmd.NetClass {
 	case "ethernet":
-		req.NetClass = control.NetDevEther
+		req.NetClass = netdetect.Ether
 	case "infiniband":
-		req.NetClass = control.NetDevInfiniband
+		req.NetClass = netdetect.Infiniband
+	default:
+		req.NetClass = control.NetDevAny
+	}
+	if cmd.AccessPoints != "" {
+		req.AccessPoints = strings.Split(cmd.AccessPoints, ",")
 	}
 
-	resp, err := control.ConfigGenerate(ctx, req)
-
+	// TODO: decide whether we want meaningful JSON output
 	if cmd.jsonOutputEnabled() {
-		return cmd.outputJSON(resp, err)
+		return cmd.outputJSON(new(control.ConfigGenerateResp), nil)
 	}
 
-	if err != nil {
+	resp, err := control.ConfigGenerate(ctx, req)
+
+	// host level errors e.g. unresponsive daos_server process
+	var bld strings.Builder
+	if err := pretty.PrintResponseErrors(resp, &bld); err != nil {
 		return err
 	}
+	cmd.log.Info(bld.String()) // no-op if no host level errors
```

Comment:
Shouldn't this be logged at Error level?

Reply:
yes it should, mind if I move this fix to my following PR? I don't think it's a blocking change